### PR TITLE
[UI] fix gmail integration tests with new oauth connection creation

### PR DESCRIPTION
**Pull request by wyvie**
This also removes unnecessary wait time before button press and semi-unnecessary wait time after button press.

//test: `@integrations-gmail`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/
  Form.java:8: Wrong order for 'org.apache.commons.lang3.BooleanUtils' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/
  CommonSteps.java:669:40: Unnecessary parentheses around expression. [UnnecessaryParentheses]
ui-tests/src/test/java/io/syndesis/qe/steps/
  CommonSteps.java:671: Empty catch block. [EmptyCatchBlock]
ui-tests/src/test/java/io/syndesis/qe/steps/
  CommonSteps.java:671:34: '|' is not followed by whitespace. [WhitespaceAround]
ui-tests/src/test/java/io/syndesis/qe/steps/
  CommonSteps.java:671:34: '|' is not preceded with whitespace. [WhitespaceAround]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:7: 'org.junit.Assert' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:8: 'org.openqa.selenium.By' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:12: Wrong order for 'io.syndesis.qe.fragments.common.list.actions.ListAction' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:15: 'lombok.extern.slf4j.Slf4j' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:17: Wrong order for 'java.util.concurrent.TimeoutException' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:36:34: '|' is not followed by whitespace. [WhitespaceAround]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:36:34: '|' is not preceded with whitespace. [WhitespaceAround]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/
  Form.java:6: 'lombok.Getter' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/
  Form.java:8: Wrong order for 'org.apache.commons.lang3.BooleanUtils' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/
  Form.java:15: Using the '.*' form of import should be avoided - java.util.*. [AvoidStarImport]
ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/
  Form.java:17: Wrong order for 'com.codeborne.selenide.Condition.exist' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:5: Wrong order for 'io.syndesis.qe.fragments.common.list.actions.ListAction' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:9: 'lombok.extern.slf4j.Slf4j' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:10: Wrong order for 'org.junit.Assert' import. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:11: 'org.openqa.selenium.By' should be separated from previous imports. [ImportOrder]
ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/
  ChooseConnectionSteps.java:15: Wrong order for 'org.junit.Assert.assertFalse' import. [ImportOrder]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Validate all credentials 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/ChooseConnectionSteps.java [hunk 1]
```diff
@@ -26,7 +32,12 @@
     @Then("^check that position of connection to fill is \"([^\"]*)\"$")
     public void verifyTypeOfConnection(String position) {
         log.info("{} connection must be active", position);
-        assertTrue("There was no active icon found for position " + position, integrationFlowView.verifyActivePosition(position));
+        try {
+            OpenShiftWaitUtils.waitFor(() -> integrationFlowView.verifyActivePosition(position),
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/ChooseConnectionSteps.java]: Maybe use `TestUtils.waitFor(...)` method?

Comment (wyvie) [reply ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/ChooseConnectionSteps.java]: Yup, you are right, it's more appropriate. Will fix in a few minutes.

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by wyvie** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by wyvie** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Validate all credentials 
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by wyvie** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Google mail Connector | Send an e-mail 
Google mail Connector | Receive an e-mail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java:678:13: '}' at column 13 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/
  finally). [RightCurly]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**